apps/cli/src/prebuild.ts

```diff
@@ -44,12 +44,12 @@ export const prebuild = async () => {
   const appRoot = "app";
 
   const routesDir = join(appRoot, "routes");
-  await rm(routesDir, { recursive: true, force: true });
-  await mkdir(routesDir, { recursive: true });
+  rm(routesDir, { recursive: true, force: true });
+  mkdir(routesDir, { recursive: true });
```

Comment:
These are callback based. Import promise based from node:fs/promises.

Reply:
yes yes, still making the changes in local 👍